pbm/prio/priority.go

```diff
@@ -111,3 +82,45 @@ func (s nodeScores) list() [][]string {
 
 	return ret
 }
+
+// CalcPriorityForAgent calculates priority for the specified agent.
+func CalcPriorityForAgent(
+	agent topo.AgentStat,
+	cfgPrio config.Priority,
+	coeffRules map[string]float64,
+) float64 {
+	if cfgPrio != nil || len(cfgPrio) > 0 {
```

Comment:
is there a different between `if cfgPrio != nil || len(cfgPrio) > 0 {` and `if len(cfgPrio) > 0 {` in logic?
I see it as the empty map gives for all nodes the same priority value (1.0). Or is there a case when we need to "disable" priority?